project/import.go

```diff
@@ -45,15 +54,66 @@ func (im *importer) Import() error {
 		WriteDebugFile(im.root, "debug/import.log", im.files)
 	}
 
+	var secrets *descope.ImportProjectSecrets
+	secrets, err := im.readSecrets(Flags.SecretsInput)
```

Comment:
there is a case where the user will not specify any secret, I think that in that case it will fail

![image](https://github.com/descope/descopecli/assets/10514677/ca8e6d7d-0329-49d7-9d97-a17ba50baab2)

do we perhaps want to make it optional?